libs/utils/mailing.py

```diff
@@ -3,7 +3,7 @@
 import codecs
 
 # TODO: Move to a constants file
-NOTIFICATION_EMAIL_FROM = "noreply@yourdomain.com"
+NOTIFICATION_EMAIL_FROM = "noreply@lanthorn.ai"
```

Comment:
Did we want to change or leave this as is? Not sure...

Reply:
My mistake!